src/System.Runtime.Extensions/tests/System/Net/WebUtility.cs

```diff
@@ -155,6 +155,9 @@ public static void HtmlEncode(string value, string expected)
             yield return Tuple.Create("%1g", "%1g");
             yield return Tuple.Create("%G1", "%G1");
             yield return Tuple.Create("%1G", "%1G");
+
+            // Double encoded
+            yield return Tuple.Create("https%3a%2f%2ffoo.com%2fbar%2fbaz%3fsignature%3dFHOywukz2tdfFpx2dPg5medckUMauhAFOuppnDx%252b2NQ%253d", "https://foo.com/bar/baz?signature=FHOywukz2tdfFpx2dPg5medckUMauhAFOuppnDx%2b2NQ%3d");
         }
```

Comment:
Why are we adding a complicated test case like this one? If we are targeting special combination, we should ideally keep just that. Or simplify it.
Real-world tests are harder to reason about when they fail :)

Reply:
@karelz good call - I put up this PR with the original String (modulo the foo/bar/baz section) to get CI to run the tests while I looked at something else. Once the tests are finished I'll push another commit to truncate the test cases to just some simpler double-encoded strings (in this case, the double encoded section is the `%252b2NQ%253d`)